ultradns/resource_ultradns_probe_http.go

```diff
@@ -1,12 +1,13 @@
 package ultradns
 
 import (
+	"errors"
 	"fmt"
-	"log"
 	"strings"
 
-	"github.com/terra-farm/udnssdk"
+	"github.com/aliasgharmhowwala/ultradns-sdk-go"
```

Comment:
No personal fork reference please.

Reply:
yup will do it once we merge the sdk plugin to master in ultradns project